addons/auth_totp/models/auth_totp.py

```diff
@@ -12,3 +16,11 @@ class AuthTotpDevice(models.Model):
     _inherit = "res.users.apikeys"
     _description = "Authentication Device"
     _auto = False
+
+    @api.autovacuum
+    def _gc_device(self):
+        self._cr.execute("""
+            DELETE FROM auth_totp_device
+            WHERE create_date < (NOW() AT TIME ZONE 'UTC' - INTERVAL '%s SECONDS')
+        """, (TRUSTED_DEVICE_AGE,))
```

Comment:
```suggestion
        """, [TRUSTED_DEVICE_AGE])
```
I personally prefer that especialy for singleton values, it's much more readable and less error-prone.